translations/en.i18n.json

```diff
@@ -6,6 +6,7 @@
       "language": "Language",
       "settings": "Settings",
       "fullscreen": "Fullscreen",
+      "fullscreenExit": "Exit fullscreen",
```

Comment:
@JonathanTGold it means we need translations for all

Reply:
Please contact Ron Liberman